<?xml version="1.0"?>

<launch>

    <node pkg="hvc_ros" type="hvc_node.py" name="hvc_node">
	    <param name="port" value="/dev/ttyUSB0" />
    </node>

</launch>
